src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -259,11 +261,13 @@ private function addExceptionToStatusSection(ArrayNodeDefinition $rootNode)
         $rootNode
             ->children()
                 ->arrayNode('exception_to_status')
-                    ->defaultValue([
-                        ExceptionInterface::class => Response::HTTP_BAD_REQUEST,
-                        InvalidArgumentException::class => Response::HTTP_BAD_REQUEST,
-                        FilterValidationException::class => Response::HTTP_BAD_REQUEST,
-                    ])
+                    ->defaultValue(
+                        array_merge([
+                            ExceptionInterface::class => Response::HTTP_BAD_REQUEST,
+                            InvalidArgumentException::class => Response::HTTP_BAD_REQUEST,
+                            FilterValidationException::class => Response::HTTP_BAD_REQUEST,
+                        ], class_exists(Version::class) ? [OptimisticLockException::class => Response::HTTP_CONFLICT] : [])
```

Comment:
using `Version::class` as that is the convention.

Reply:
Looks useless, PHP will allow `OptimisticLockException::class` even if the class doesn't actually exist. It doesn't hurt to have this extra key.

Reply:
done